src/utils/cronJobs.ts

```diff
@@ -8,9 +8,9 @@ export function startCronJobs() {
   let isHealthCheckRunning = false
 
   /**
-   * try to start fetchBlockchainData every minute
+   * try to start fetchBlockchainData every 15 minutes
    */
-  cron.schedule('* * * * *', async () => {
+  cron.schedule('2,17,32,47 * * * *', async () => {
```

Comment:
thought(non-blocking): It's easy to read so that's why I would keep your solution. If you are a cronjob guru though it looks like there is also a different way to write this https://stackoverflow.com/a/19204734/5639677 

Might not work with node-cron though.